source/extensions/omni.isaac.lab_tasks/config/extension.toml

```diff
@@ -22,7 +22,7 @@ requirements = [
     "stable-baselines3>=2.1",
     "rl-games==1.6.1",
     "rsl-rl@git+https://github.com/leggedrobotics/rsl_rl.git",
-    "skrl>=1.1.0"
+    "skrl>=1.2.0"
```

Comment:
Should we keep this as an == ?

Reply:
Mmmn, `>=` is okay... I am not planning breaking changes for `skrl`... but up to you